recipes/helios-menuinst/run_test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if ! test -f $PREFIX/Menu/helios.json; then
```

Comment:
You can probably get by by adding this directly in meta.yaml as:

```
test -f $PREFIX/Menu/helios.json
```

Reply:
I did something like that and it executed with `sh` instead of `bash` and produced an error.

Reply:
Hm, weird. That's standard posix 🤔 Can you try again so we can see the traceback?